rcl_logging_spdlog/test/test_logging_interface.cpp

```diff
@@ -107,7 +86,7 @@ TEST_F(LoggingTest, init_failure)
   rcutils_reset_error();
 
   // Force failure to create directories
-  fs::path fake_home = current_path() / "fake_home_dir";
+  fs::path fake_home = rcpputils::fs::current_path() / "fake_home_dir";
```

Comment:
Can we make this:

```suggestion
  rcpputils::fs::path fake_home = rcpputils::fs::current_path() / "fake_home_dir";
```

And then remove the `namespace fs = rcpputils::fs;` at the top of the file?  I guess this would also require other changes further below, but I think that would be nicer.